go.mod

```diff
@@ -10,12 +10,12 @@ require (
 	github.com/jmoiron/sqlx v1.2.0
 	github.com/lib/pq v1.0.0
 	github.com/makerdao/vdb-transformer-utilities v0.0.0-20201021185605-2af87288ff1b
-	github.com/makerdao/vulcanizedb v0.1.0
+	github.com/makerdao/vulcanizedb v0.1.1-0.20210120155730-fed8d6aead03
 	github.com/mitchellh/mapstructure v1.1.2
 	github.com/onsi/ginkgo v1.14.1
 	github.com/onsi/gomega v1.10.2
 	github.com/sirupsen/logrus v1.7.0
-	github.com/spf13/cobra v0.0.3
+	github.com/spf13/cobra v0.0.5
```

Comment:
Were these updates intentional? go.mod likes to update itself for me, so I'm paranoid.

Reply:
I intentionally updated the vulcanizedb version so that we'd get the new pending status from https://github.com/makerdao/vulcanizedb/blob/prod/libraries/shared/storage/diff_repository.go#L43, and then I think that the cobra update followed from that.

Updating vdb probably wasn't totally necessary because the only place this PR relies on the new pending status in code (vs in the db) is in tests, so I'd be happy to not check this in/